### PR TITLE
Ot 1526 add varying focal length to keentools core

### PR DESCRIPTION
**Pull request by amilovsky**

### CODE REVIEW

File: keentools_facebuilder/utils/materials.py [hunk 1]
```diff
@@ -237,10 +237,39 @@ def progress_callback(progress):
...
-    built_texture = tb.build_texture(
-        frames_count, frame_data_loader, progress_callback)
-    bpy.context.window_manager.progress_end()
+    #-------------- New baking
+    # bpy.context.window_manager.progress_begin(0, 1)
+    # built_texture = tb.build_texture(
+    #     frames_count, frame_data_loader, progress_callback)
+    # bpy.context.window_manager.progress_end()
+
+    #-------------- Old baking
```
Comment (SergeyKrivohatskiy) [inline keentools_facebuilder/utils/materials.py:246]: Why is that change in pull request? frame_data_loader should be changed instead

File: keentools_facebuilder/viewport.py [hunk 2]
```diff
@@ -348,7 +353,7 @@ def update_residuals(cls, fb, context, headobj, keyframe):
 
         # ----------
         # Projection
-        projection = fb.projection_mat().T
+        projection = fb.projection_mat(keyframe).T  # Fix
```
Comment (SergeyKrivohatskiy) [inline keentools_facebuilder/viewport.py:356]: What is this comment about?

File: keentools_facebuilder/utils/materials.py (hunk 1 above)
Comment (amilovsky) [reply keentools_facebuilder/utils/materials.py:246]: We have no new texture builder in pykeentools within this branch. So after a successful merge, we just need to delete the old baking and uncomment the new one.

File: keentools_facebuilder/viewport.py (hunk 2 above)
Comment (amilovsky) [reply keentools_facebuilder/viewport.py:356]: old comment, just forgot about it

**Pull request merged**